lib/util.js

```diff
@@ -899,3 +899,14 @@ exports._exceptionWithHostPort = function(err,
   }
   return ex;
 };
+
+
+exports.decorateErrorStack = function(err) {
```

Comment:
Yes. I tried using `internal/util`, but the REPL doesn't seem to like it. It gives this error when starting node:

`Error: Cannot find module 'internal/util'`